bodhi/tests/client/test_bindings.py

```diff
@@ -24,6 +25,435 @@
+
+    def test_with_package_el_build(self):
+        """
+        Test with the package arg expressed as an el7 build.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(package='bodhi-2.4.0-1.el7')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'builds': 'bodhi-2.4.0-1.el7'})
+
+    def test_with_package_epel_id(self):
+        """
+        Test with the package arg expressed as a EPEL update id.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(package='FEDORA-EPEL-2017-c3b112eb9e')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'updateid': 'FEDORA-EPEL-2017-c3b112eb9e'})
+
+    def test_with_package_fc_build(self):
+        """
+        Test with the package arg expressed as a fc26 build.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(package='bodhi-2.4.0-1.fc26')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'builds': 'bodhi-2.4.0-1.fc26'})
+
+    def test_with_package_fedora_id(self):
+        """
+        Test with the package arg expressed as a Fedora update id.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(package='FEDORA-2017-52506b30d4')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'updateid': 'FEDORA-2017-52506b30d4'})
+
+    def test_with_package_name(self):
+        """
+        Test with the package arg expressed as a package name.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(package='bodhi')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'packages': 'bodhi'})
+
+    def test_with_release(self):
+        """
+        Test with a 'release' kwarg.
+        """
+        client = bindings.BodhiClient()
+        client.send_request = mock.MagicMock(return_value='return_value')
+
+        result = client.query(packages='bodhi', release='f26')
+
+        self.assertEqual(result, 'return_value')
+        client.send_request.assert_called_once_with(
+            'updates/', verb='GET', params={'packages': 'bodhi', 'releases': 'f26'})
```

Comment:
Should it be wrapping the value of 'releases' in a list if it's not already a list? I just noticed the docblock says it can be a list (and its name implies that) so I wondered.